iphone/Maps/LocalizedStrings/ru.lproj/Localizable.stringsdict

```diff
@@ -19,11 +16,8 @@
 			<string>%d метка</string>
 			<key>few</key>
 			<string>%d метки</string>
-			<key>other</key>
-			<string>%d меток</string>
```

Comment:
Why was it deleted? :/

Reply:
https://github.com/organicmaps/organicmaps/pull/10321